client_src/js/services/gocde-scripts.service.js

```diff
@@ -1,5 +1,7 @@
-import UI from "../functions/ui.js";
-import OctoFarmClient from "../../services/octofarm-client.service";
+import UI from "../lib/functions/ui.js";
+import OctoFarmClient from "./octofarm-client.service";
+
+//TODO: Rework in another PR
```

Comment:
Personal preference: add a User Story for small refactors and a Task to refactor this service file.
I have a tendency add TODO as well, so its a more of a thing I would like to stop doing myself.